src/bundle/Resources/views/content/tab/versions/tab.html.twig

```diff
@@ -47,6 +47,47 @@
             </div>
         {% endif %}
     </section>
+    <div class="modal fade ez-modal ez-modal--version-conflict" id="version-conflict-modal" tabindex="-1" role="dialog">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5>{{ 'version.conflict.header'|trans|desc('Version conflict when editing') }}</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <p class="ez-modal-body__explanation">
+                        <strong>
+                            {{ 'version.conflict.are_you_sure'|trans|desc(
+                            'Are you sure you want to continue and overwrite the latest version?'
+                            ) }}
+                        </strong>
+                    </p>
+                    <p class="ez-modal-body__explanation">
+                        {{ 'version.conflict.consequences'|trans|desc(
+                        'If you continue, this version will overwrite.'
+                        ) }}
+                    </p>
+                    <p class="ez-modal-body__explanation">
+                        {{ 'version.conflict.choices'|trans|desc(
+                        'To resolve the conflict, click on Cancel and delete the draft. Then edit the published content item.'
+                        ) }}
+                    </p>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">
+                        {{ 'version.conflict.form.cancel'|trans|desc('Cancel') }}
+                    </button>
+                    <a id="edit-conflicted-draft" class="btn btn-primary"
+                       href="">
```

Comment:
Not an HTML expert but I don't think empty `href` attribute creates valid HTML element. How about using `#` as a default value so at least it has a meaning?